config/software/libarchive.rb

```diff
@@ -28,20 +28,19 @@
 build do
   env = with_standard_compiler_flags(with_embedded_path)
```

Comment:
Double check if you need `with_standard_compiler_flags(with_embedded_path, bfd_flags: true)`.  Otherwise, it might compile but fail to link later because of architecture mismatch.